manage_breast_screening/clinics/jinja2/index.html

```diff
@@ -48,32 +31,28 @@ <h1>{{ presenter.heading }}</h1>
       <td>
         <a href="/clinics/{{ clinic.id }}" class="nhsuk-link">
           {{ clinic.location_name }}
-          <br>
+          <br />
           ({{ clinic.session_type }})
         </a>
       </td>
-      <td>{{ clinic.starts_at | no_wrap }}<br>
+      <td>
+        {{ clinic.starts_at | no_wrap }}<br />
         {{clinic.time_range | as_hint }}
       </td>
       <td>
         {{ clinic.type }}
-        <br>
+        <br />
```

Comment:
this is not the style I am expecting for html5 - self closing tags should not be necessary